pygmt/tests/test_grdimage.py

```diff
@@ -41,6 +41,14 @@ def test_grdimage_file():
     return fig
 
 
+@pytest.mark.mpl_image_compare
+def test_grdimage_rgb_files():
+    "Plot an image using Red, Green, and Blue file inputs"
+    fig = Figure()
+    fig.grdimage(grid=["@earth_relief_60m", "@earth_relief_60m", "@earth_relief_60m"])
```

Comment:
I don't quite this unit test example, since they're clearly not RGB grids. Does anyone have a better sample dataset recommendation? Preferably a small one that has a fairly stable URL.

Reply:
@weiji14 ideally, we should use a dataset that is open-access and published somewhere (figshare, zenodo, etc) or in the public domain. Maybe @djhoese can help us out here :slightly_smiling_face:

Reply:
Hm I can only think of raw data at the moment. What format do you need as input (I'm not super familiar with the limitations of gmt)?

Reply:
GeoTiffs would be nice. Or anything GDAL can read I think. I quite like the zenodo idea, something with a persistent doi. But happy with any other good public domain ones too!

Reply:
There has to be some landsat data or something. @snowman2 does a lot with rioxarray and may have an idea of a persistent/versioned RGB geotiff. I'll ask the Pytroll core developers too.

Reply:
If I understand what you are asking, there is an RGB tif in the example here: https://corteva.github.io/rioxarray/html/examples/COG.html

Reply:
I did look at the rioxarray example a while ago actually, but the test here requires tiffs of individual bands, e.g r.tif, g.tif, b.tif. Landsat on AWS does provide individual bands but they're quite big...

Reply:
Oh if you need the bands to be in separate tif files then that is going to be harder to track down.

Reply:
You could just take a regular RGB file and write each band to its own tif file.

```python
import rioxarray
rds = rioxarray,open_rasterio(....)
rdssub = rds.sel(band=1)
rdssub,rio.to_raster(....)
```

Reply:
Yeah, I thought I might need to create some dummy test files. @seisman just mentioned a similar idea with splitting a raster into R,G,B bands. Just seems like quite a bit of work for a unit test (requires set-up and tear down).